servers/cu/src/domain/client/metrics.js

```diff
@@ -48,10 +75,16 @@ export const gaugeWith = ({ prefix = 'ao_cu' } = {}) => {
         ? { collect: async function () { this.set(await collect()) } }
         : {}
       ),
+      labelNames,
       enableExemplars: true
     })
 
     return {
+      labels: (labels) => ({
+        inc: (n) => g.labels(labels).inc(n),
+        dec: (n) => g.labels(labels).dec(n),
+        set: (n) => g.labels(labels).set(n)
+      }),
```

Comment:
Instead of a separate API, should `labels` just be passed as a second optional parameter for the existing apis? Then the impl can decide whether to set labels or not.